text/0000-parse-result-meta-data.md

```diff
@@ -0,0 +1,78 @@
+- Start Date: 2015-11-20
+- RFC PR: (leave this empty)
+- Refract Issue: (leave this empty)
+
+# Summary
+
+This RFC outlines ideas in adding meta data into into elements through the
+means of meta links and adding link relations to the Parse Result namespace.
+
+# Motivation
+
+Parsing happens in multiple steps and information is added by different layers
+in the stack in some situations. It would be helpful for end users to have meta
+data to let the user know where data came from and which information was added
+that did not exist in the original representation.
+
+The idea with proposing meta links would be to provide a generic way to link to
+out-of-band information to allow the user to explore more information in a
+given context.
+
+# Detailed design
+
+The proposal is to add a `links` property to `meta` and propose two link
+relations for the Parse Result namespace.
+
+## Links Property in Meta
+
+The property would be defined as the following the `meta` object of all
+elements. This would be added in the Refract base spec.
+
+- `links` (array)
+    - (object)
+        - `rel` - Link relation of the link
+        - `href` - Resolvable URL for the link
+
+The `rel` property MAY be any defined [link
+relation](http://www.iana.org/assignments/link-relations/link-relations.xhtml)
+or a resolvable URL to documentation on the link relation.
+
+## Parse Result Link Relations
+
+### `inferred`
+
+The `inferred` link relation defines a link to documentation on how an element
```

Comment:
To go to the world of hypermedia representors, say you were to parse Collection+JSON to a canonical model. There are many situations where you would either drop information or make assumptions/inferences. This would be a link from the elements that have been inferred to a document describing why this is so and what was inferred.

Of course, a client accessing this document can tell if an element has been inferred by the presence of this link relation, but it can also access additional information if need be through the link.